cub/cub/iterator/discard_output_iterator.cuh

```diff
@@ -95,11 +95,6 @@ public:
 private:
   OffsetT offset;
 
-#if defined(_WIN32) || !defined(_WIN64)
-  // Workaround for win32 parameter-passing bug (ulonglong2 argmin DeviceReduce)
-  OffsetT pad[CUB_MAX(1, (16 / sizeof(OffsetT) - 1))] = {};
-#endif
```

Comment:
The same workaround also exists in `ConstantInputIterator`: https://github.com/NVIDIA/cccl/blob/25bd198f3c347673dcfd8bb35c1b460f2dd40085/cub/cub/iterator/constant_input_iterator.cuh#L131-L134

But there it only says `#ifdef _WIN32`, so the workaround is also active on 64bit Windows. Does anybody know whether we can remove that workaround as well?

Reply:
The workaround was introduced in 2013 by 24b99892. Discussing with @miscco offline, let's drop it and see what the CI says.